Sources/Amplitude/Amplitude.m

```diff
@@ -284,6 +280,82 @@ - (instancetype)initWithInstanceName:(NSString *)instanceName {
     return self;
 }
 
+- (NSString *)getDataStorageKey:(NSString *)key {
+    NSString *dataStorageKey = [NSString stringWithFormat:@"%s_%@_%@_%@", "amplitude", _bundleIdentifier ,self.instanceName, key];
```

Comment:
nit: the space in `_bundleIdentifier ,self.instanceName`

Reply:
Make it to be a static func

Reply:
My main concern for having this function is that other folks might not know to call this at all. However, I have no better solution atm.